plugin-server/src/worker/ingestion/hooks.ts

```diff
@@ -85,33 +129,34 @@ export function getValueOfToken(
         } else {
             const propertyName = `$${tokenParts[1]}`
             const property = event.properties?.[propertyName]
-            text = stringify(property)
-            markdown = text
+            markdown = text = webhookEscape(property, webhookType)
         }
     } else if (tokenParts[0] === 'person') {
         if (tokenParts.length === 1) {
             ;[text, markdown] = getUserDetails(event, siteUrl, webhookType)
         } else if (tokenParts[1] === 'properties' && tokenParts.length > 2) {
             const propertyName = tokenParts[2]
             const property = event.person_properties?.[propertyName]
-            text = stringify(property)
-            markdown = text
+            markdown = text = webhookEscape(property, webhookType)
         }
     } else if (tokenParts[0] === 'action') {
         if (tokenParts[1] === 'name') {
             ;[text, markdown] = getActionDetails(action, siteUrl, webhookType)
         }
     } else if (tokenParts[0] === 'event') {
-        if (tokenParts[1] === 'name') {
-            text = stringify(event.event)
+        if (tokenParts[1] === 'uuid') {
+            markdown = text = webhookEscape(event.eventUuid, webhookType)
+        } else if (tokenParts[1] === 'name') {
+            markdown = text = webhookEscape(event.event, webhookType)
+        } else if (tokenParts[1] === 'event') {
+            ;[text, markdown] = getEventDetails(event, siteUrl, webhookType)
```

Comment:
I think we should use the same approach as with `person` here. So `[event]` should be the linkified version, while `[event.event]` should return the name (and we probably should not use `[event.name]`, since in e.g. HogQL events have an `event` column and not `name`

Reply:
also makes sense. changed